src/app/services/resource.service.ts

```diff
@@ -35,7 +35,9 @@ import {TransmartTrialVisit} from '../models/transmart-models/transmart-trial-vi
 import {CategoricalAggregate} from '../models/aggregate-models/categorical-aggregate';
 import {TransmartResourceService} from './transmart-services/transmart-resource.service';
 
-@Injectable()
+@Injectable({
+  providedIn: 'root',
```

Comment:
Why this change, if we still have ResourceService specified in providers of root module? Why only for this service?

Reply:
indeed, single services can be moved out of appModule